### ISSUE TITLE
Another plugin with the same functionality

### ISSUE THREAD

**Issue by jerviscui** [zfei/obsidian-url-namer#1]
I'm using another plugin with the same functionality.
https://github.com/zolrath/obsidian-auto-link-title

I wonder what the difference is?

**Comment by zfei** [comment]
@jerviscui thanks for letting me know. Unfortunately, I wasn't able to find this plugin when I first look for such functionality, and it does look like I reinvented a wheel. The plugin you pointed out does look pretty well maintained. Will probably think of PR'ing some new ideas I was thinking about into it and retire mine.

Literally speaking, the difference is mine can process a _selected text block_ while theirs can only process a single link where the cursor's at; mine _doesn't handle realtime paste and mobile compatibility_ while theirs do. Another main difference is, many sites nowadays do _lazy-loaded html_, i.e. only a default title tag is provided in the html file, the actual title and content is loaded by js on the fly. I added a custom logic for Wechat articles and I intended to make such logic configurable.
